flink-core/src/test/java/org/apache/flink/api/common/operators/base/FlatMapOperatorCollectionTest.java

```diff
@@ -90,13 +82,13 @@ private void testExecuteOnCollection(
                                         taskInfo,
                                         null,
                                         executionConfig,
-                                        new HashMap<String, Future<Path>>(),
-                                        new HashMap<String, Accumulator<?, ?>>(),
+                                        new HashMap<>(),
+                                        new HashMap<>(),
                                         UnregisteredMetricsGroup.createOperatorMetricGroup()),
                                 executionConfig);
 
-        Assert.assertEquals(input.size(), result.size());
-        Assert.assertEquals(input, result);
+        assertThat(result).hasSize(input.size());
+        assertThat(result).isEqualTo(input);
```

Comment:
```suggestion
        assertThat(result).hasSameSizeAs(input).isEqualTo(input);
```
or just isEqualTo for short